src/app/main/home/detail/detail.component.ts

```diff
@@ -31,6 +31,8 @@ export class DetailComponent implements OnInit {
   protected galleryImages: GalleryImage[];
   protected dark;
   protected detail;
+  templateURLReadme: string;
+  teplateURLGithub: string;
```

Comment:
```suggestion
  templateURLGithub: string;
```
Replace variable name